how-to/run-kata-with-k8s.md

```diff
@@ -1,64 +1,86 @@
 # Run Kata Containers with Kubernetes
 
-* [Pre-requisites](#pre-requisites)
 * [Run Kata Containers with Kubernetes](#run-kata-containers-with-kubernetes)
+  * [Prerequisites](#prerequisites)
   * [Install a CRI implementation](#install-a-cri-implementation)
      * [CRI-O](#cri-o)
+        * [Kubernetes Runtime Class (CRI-O v1.12 )](#kubernetes-runtime-class-cri-o-v112)
+        * [Untrusted annotation (until CRI-O v1.12)](#untrusted-annotation-until-cri-o-v112)
+        * [Network namespace management](#network-namespace-management)
      * [containerd with CRI plugin](#containerd-with-cri-plugin)
   * [Install Kubernetes](#install-kubernetes)
      * [Configure for CRI-O](#configure-for-cri-o)
      * [Configure for containerd](#configure-for-containerd)
   * [Run a Kubernetes pod with Kata Containers](#run-a-kubernetes-pod-with-kata-containers)
 
-## Pre-requisites
-This guide requires Kata Containers available on your system, and it can be installed
-following [this guide](https://github.com/kata-containers/documentation/blob/master/install/README.md).
-
+## Prerequisites
+This guide requires Kata Containers available on your system, install-able by following [this guide](https://github.com/kata-containers/documentation/blob/master/install/README.md).
```

Comment:
I wonder if we should point this at [kata-deploy](https://github.com/kata-containers/packaging/tree/master/kata-deploy), as:
- this is afaik the recommended way to install kata on k8s clusters
- you don't actually need kata installed on the master node (if it is not schedulable), or the host (where you run `kubectl` for instance), just the schedulable nodes that will run pods...